src/HelloSignalR/Startup.cs

```diff
@@ -14,10 +20,76 @@ public void ConfigureServices(IServiceCollection services)
         public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
             app.UseStaticFiles();
-            app.UseSignalR(routes =>
+            app.UseSignalRWithSpecification(routes =>
```

Comment:
All due respect, i appreciate your consistency here but I am concerned mixing the actual SignalR wire-up and the specification/description in the same breath might confuse some. For instance, what if someone wants to wire up the Azure SignalR Service here? Would you add pass-throughs here to enable folks to do this, too?

Reply:
Yes, i dont like this too, that’s why i’m asking for an api explorer... without that there are only two options: This interceptor/wrapper approach or specifing the hubs twice (one for the spec and one for signalr)... this PR is just a prototype/my sample impl

Reply:
For core 3.0 this can now be implemented, see also

https://github.com/dotnet/aspnetcore/pull/8425